frontend/src/app/shared/components/full-calendar-wrapper/full-calendar-wrapper.component.ts

```diff
@@ -82,6 +100,38 @@ export class FullCalendarWrapperComponent implements OnInit, OnDestroy {
       });
   }
 
+  handleDateSet(datesSetArgs : DatesSetArg) {
+    this.getEventsForCurrentView(datesSetArgs)
```

Comment:
Ha csak itt van meghívva ez a fügvény nyugodtan ki lehet törölni és a tartalmát berakhatod a handleDateSet-be. Így könnyebb lesz tesztelni.